2013/01/07 22:25:24 [error] [exception.CException] exception 'CException' with message 'El asset "C:\xampp\htdocs\sgt\protected/vendors/jqueryui/ThemeSGT/ a ser publicado no existe.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:257
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\layouts\main.php(19): CAssetManager->publish('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\sgt\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/sgt/index.php/site/index
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/07 22:26:40 [error] [exception.CException] exception 'CException' with message 'El asset "C:\xampp\htdocs\sgt\protected\vendors\jqueryui\ThemeSGT a ser publicado no existe.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:257
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\layouts\main.php(19): CAssetManager->publish('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\sgt\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/sgt/index.php/site/index
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/07 22:28:51 [error] [exception.CException] exception 'CException' with message 'El asset "C:\xampp\htdocs\sgt\protected\vendors\jqueryui\ThemeSGT\  a ser publicado no existe.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:257
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\layouts\main.php(19): CAssetManager->publish('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\sgt\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/sgt/index.php/site/index
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/07 22:29:25 [error] [exception.CException] exception 'CException' with message 'El asset "C:\xampp\htdocs\sgt\protected\vendors\jquery.ui\ThemeSGT\  a ser publicado no existe.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:257
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\layouts\main.php(19): CAssetManager->publish('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\sgt\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/sgt/index.php/site/index
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/07 22:29:42 [error] [exception.CException] exception 'CException' with message 'El asset "C:\xampp\htdocs\sgt\protected/vendors/jquery.ui/ThemeSGT/ a ser publicado no existe.' in C:\xampp\htdocs\yii\framework\web\CAssetManager.php:257
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\layouts\main.php(19): CAssetManager->publish('C:\xampp\htdocs...')
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#5 C:\xampp\htdocs\yii\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(206): COutputProcessor->run()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#8 C:\xampp\htdocs\sgt\protected\views\layouts\column1.php(6): CBaseController->endContent()
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(784): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(32): CController->render('index')
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('index')
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/index')
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#21 {main}
REQUEST_URI=/sgt/index.php/site/index
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:39:33 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:39:48 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:40:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "images" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('images')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('images')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/images/tal...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:51:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "images/taller/taller2.jpg"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('images/taller/t...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "images/taller/taller.jpg"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('images/taller/t...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "images/taller/taller3.jpg"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('images/taller/t...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "images/taller/taller3.jpg"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('images/taller/t...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/images/taller/taller3.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "images/taller/taller.jpg"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('images/taller/t...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/images/taller/taller.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "images/taller/taller2.jpg"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('images/taller/t...')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/images/taller/taller2.jpg
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:52:52 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:56:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:56:06 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:56:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:56:20 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:57:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:57:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:57:09 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:57:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/close....')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/close.png
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/08 23:57:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "img" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('img')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('img')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/img/loadin...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/img/loading.gif
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/09 00:09:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/09 00:11:31 [error] [php] mail(): Failed to connect to mailserver at &quot;localhost&quot; port 25, verify your &quot;SMTP&quot; and &quot;smtp_port&quot; setting in php.ini or use ini_set() (C:\xampp\htdocs\sgt\protected\controllers\SiteController.php:67)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/contact
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (67)
in C:\xampp\htdocs\sgt\index.php (13)
2013/01/09 01:08:57 [error] [php] Undefined variable: model (C:\xampp\htdocs\sgt\protected\components\UserIdentity.php:20)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/login
in C:\xampp\htdocs\sgt\protected\components\UserIdentity.php (20)
in C:\xampp\htdocs\sgt\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (94)
2013/01/09 01:10:15 [error] [php] Undefined variable: model (C:\xampp\htdocs\sgt\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/login
in C:\xampp\htdocs\sgt\protected\components\UserIdentity.php (23)
in C:\xampp\htdocs\sgt\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (94)
2013/01/09 01:11:01 [error] [php] Undefined variable: model (C:\xampp\htdocs\sgt\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/login
in C:\xampp\htdocs\sgt\protected\components\UserIdentity.php (23)
in C:\xampp\htdocs\sgt\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (94)
2013/01/09 01:14:26 [error] [php] Undefined variable: model (C:\xampp\htdocs\sgt\protected\components\UserIdentity.php:23)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/login
in C:\xampp\htdocs\sgt\protected\components\UserIdentity.php (23)
in C:\xampp\htdocs\sgt\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (94)
2013/01/09 01:16:31 [error] [php] Undefined variable: model (C:\xampp\htdocs\sgt\protected\components\UserIdentity.php:24)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\validators\CValidator.php(213): CInlineValidator->validateAttribute()
#1 C:\xampp\htdocs\yii\framework\base\CModel.php(159): CInlineValidator->validate()
#2 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(94): LoginForm->validate()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionLogin()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/login
in C:\xampp\htdocs\sgt\protected\components\UserIdentity.php (24)
in C:\xampp\htdocs\sgt\protected\models\LoginForm.php (52)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (94)
2013/01/09 21:27:59 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php
---
2013/01/09 21:36:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "contacto.php" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('contacto.php')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('contacto.php')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/contacto.p...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/site/contacto.php
HTTP_REFERER=http://localhost/sgt/index.php/site/page?view=about
---
2013/01/09 21:58:05 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'No es posible resolver la solicitud "consulta"' in C:\xampp\htdocs\yii\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('consulta')
#1 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#2 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/sgt/index.php/consulta
---
2013/01/09 21:58:48 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('consultas/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/sgt/index.php/consultas/admin
HTTP_REFERER=http://localhost/sgt/index.php/consultas/create
---
2013/01/09 22:33:37 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('admin')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('consultas/admin')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/sgt/index.php/consultas/admin
HTTP_REFERER=http://localhost/sgt/index.php/consultas/index
---
2013/01/09 23:40:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php
---
2013/01/12 01:58:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php
---
2013/02/06 20:25:30 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "/sgt/index.php/Clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('/sgt/index.php/...', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 20:25:46 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "/sgt/index.php/clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('/sgt/index.php/...', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 20:25:48 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "/sgt/index.php/clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('/sgt/index.php/...', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 20:27:42 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('clientes/create', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 20:29:13 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "/sgt/index.php/protected/clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('/sgt/index.php/...', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 20:29:22 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('clientes/create', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 20:29:37 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "Clientes/create" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(41): CController->renderPartial('Clientes/create', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#3 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): CController->renderPartial('_form', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#9 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(79): CController->render('create', Array)
#10 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#21 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#22 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#23 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/06 21:27:53 [error] [php] Undefined variable: model_cliente (C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php:230)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): OrdenesController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(83): OrdenesController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php (230)
in C:\xampp\htdocs\sgt\protected\views\ordenes\create.php (20)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (83)
2013/02/06 21:29:24 [error] [php] Undefined variable: model_cliente (C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php:230)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): OrdenesController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(83): OrdenesController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php (230)
in C:\xampp\htdocs\sgt\protected\views\ordenes\create.php (20)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (83)
2013/02/06 21:30:07 [error] [php] Undefined variable: model_cliente (C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php:230)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(20): OrdenesController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(83): OrdenesController->render()
#7 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#16 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#18 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#19 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php (230)
in C:\xampp\htdocs\sgt\protected\views\ordenes\create.php (20)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (83)
2013/02/06 21:39:17 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/06 21:48:57 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/06 21:55:47 [error] [exception.CHttpException.403] exception 'CHttpException' with message 'Usted no se encuentra autorizado a realizar esta acción.' in C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php:179
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\auth\CAccessControlFilter.php(142): CAccessControlFilter->accessDenied(Object(CWebUser), 'Usted no se enc...')
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(38): CAccessControlFilter->preFilter(Object(CFilterChain))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#11 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/06 21:56:27 [error] [exception.CException] exception 'CException' with message 'OrdenesController y sus behaviors no tienen un método o closure llamado "actioncreaClienteRap".' in C:\xampp\htdocs\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CComponent->__call('actioncreaClien...', Array)
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actioncreaClienteRap()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/06 21:57:01 [error] [exception.CException] exception 'CException' with message 'OrdenesController y sus behaviors no tienen un método o closure llamado "actioncreaClienteRap".' in C:\xampp\htdocs\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CComponent->__call('actioncreaClien...', Array)
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actioncreaClienteRap()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/06 21:57:48 [error] [exception.CException] exception 'CException' with message 'OrdenesController y sus behaviors no tienen un método o closure llamado "actioncreaClienteRap".' in C:\xampp\htdocs\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): CComponent->__call('actioncreaClien...', Array)
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actioncreaClienteRap()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/06 21:59:20 [error] [php] Undefined index: admin (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:194)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (194)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/06 22:00:39 [error] [exception.CException] exception 'CException' with message 'Clientes y sus behaviors no tienen un método o closure llamado "AllConcat".' in C:\xampp\htdocs\yii\framework\base\CComponent.php:265
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(225): CComponent->__call('AllConcat', Array)
#1 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(198): CActiveRecord->__call('AllConcat', Array)
#2 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(198): Clientes->AllConcat()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreaClienteRap()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('creaClienteRap')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/creaCli...')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sgt/index.php/ordenes/creaClienteRap
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/07 21:10:18 [error] [php] Undefined index: tipo (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:210)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/creaEquipoRap
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (210)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/07 21:15:48 [error] [exception.CException] exception 'CException' with message 'Propiedad "Ordenes"."id" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id')
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\view.php(8): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): CController->render('view', Array)
#7 [internal function]: OrdenesController->actionView('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/07 21:17:59 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/07 21:18:21 [error] [exception.CException] exception 'CException' with message 'Propiedad "Ordenes"."id" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id')
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\update.php(9): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('update', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(107): CController->render('update', Array)
#7 [internal function]: OrdenesController->actionUpdate('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/update/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/07 21:20:06 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/07 21:20:46 [error] [php] Undefined variable: model_cliente (C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php:263)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\update.php(23): OrdenesController->renderPartial()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(107): OrdenesController->render()
#7 unknown(0): OrdenesController->actionUpdate()
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/update?id=254
in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php (263)
in C:\xampp\htdocs\sgt\protected\views\ordenes\update.php (23)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (107)
2013/02/07 21:20:48 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/07 21:23:10 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/07 21:24:51 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/view?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/07 21:31:06 [error] [php] Undefined variable: model_cliente (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:107)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/update/254
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (107)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/07 21:31:46 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/view?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/update/254
---
2013/02/07 21:33:10 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/view?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/07 21:45:07 [error] [exception.CException] exception 'CException' with message 'El atributo debe especificarse en el formato "Nombre:Tipo:Etiqueta", donde "Tipo" y "Etiqueta" son opcionales.' in C:\xampp\htdocs\yii\framework\zii\widgets\CDetailView.php:180
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CDetailView->run()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\view.php(41): CBaseController->widget('zii.widgets.CDe...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): CController->render('view', Array)
#7 [internal function]: OrdenesController->actionView('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/07 21:47:27 [error] [exception.CException] exception 'CException' with message 'El atributo debe especificarse en el formato "Nombre:Tipo:Etiqueta", donde "Tipo" y "Etiqueta" son opcionales.' in C:\xampp\htdocs\yii\framework\zii\widgets\CDetailView.php:180
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(173): CDetailView->run()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\view.php(41): CBaseController->widget('zii.widgets.CDe...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): CController->render('view', Array)
#7 [internal function]: OrdenesController->actionView('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/07 23:24:02 [error] [exception.CException] exception 'CException' with message 'OrdenesController no ha podido encontrar la vista "/equipo/view" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(258): CController->renderPartial('/equipo/view', Array)
#1 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionMuestraEquipo()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('muestraEquipo')
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('Ordenes/muestra...')
#12 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#13 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#14 {main}
REQUEST_URI=/sgt/index.php/Ordenes/muestraEquipo
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/13 20:17:03 [error] [exception.CException] exception 'CException' with message 'Propiedad "Ordenes"."service_oficial" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('service_oficial')
#1 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(2255): CActiveRecord->__get('service_oficial')
#2 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(1506): CHtml::resolveValue(Object(Ordenes), 'service_oficial')
#3 C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php(747): CHtml::activeCheckBox(Object(Ordenes), 'service_oficial', Array)
#4 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(287): CActiveForm->checkBox(Object(Ordenes), 'service_oficial', Array)
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(22): CController->renderPartial('_form', Array)
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#10 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#12 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#13 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(89): CController->render('create', Array)
#14 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#17 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#21 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#22 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#24 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#25 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#26 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#27 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/13 20:17:57 [error] [php] Missing argument 2 for CHtml::label(), called in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php on line 286 and defined (C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php:546)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#2 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(22): OrdenesController->renderPartial()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#7 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(89): OrdenesController->render()
#8 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#10 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#13 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#16 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#17 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#19 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#20 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php (286)
in C:\xampp\htdocs\sgt\protected\views\ordenes\create.php (22)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (89)
2013/02/13 20:20:12 [error] [php] Object of class Ordenes could not be converted to string (C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php:1918)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\helpers\CHtml.php(740): inputField()
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(287): checkBox()
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#5 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(22): OrdenesController->renderPartial()
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require()
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): OrdenesController->renderInternal()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#10 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(89): OrdenesController->render()
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#16 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#17 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#18 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#21 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#22 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php (287)
in C:\xampp\htdocs\sgt\protected\views\ordenes\create.php (22)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (89)
2013/02/13 20:23:24 [error] [exception.CException] exception 'CException' with message 'Propiedad "Ordenes"."service_oficial" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('service_oficial')
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(287): CActiveRecord->__get('service_oficial')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(22): CController->renderPartial('_form', Array)
#6 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#7 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#8 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#9 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#10 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(89): CController->render('create', Array)
#11 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#13 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#14 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#15 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#17 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#19 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#21 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#22 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#23 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/13 20:27:05 [error] [php] Undefined index: service_oficial (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:76)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (76)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 20:28:06 [error] [system.db.CDbCommand] CDbCommand::execute() falló: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '254' for key 'PRIMARY'. La sentencia SQL ejecutada fue: INSERT INTO `ordenes` (`nro_orden`, `id_cliente`, `id_equipo`, `nro_serie`, `adquirido_en`, `nro_factura`, `falla`, `reparacion`, `estado`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8).
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (75)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 20:28:06 [error] [exception.CDbException] exception 'CDbException' with message 'CDbCommand falló al ejecutar la sentencia SQL: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '254' for key 'PRIMARY'. The SQL statement executed was: INSERT INTO `ordenes` (`nro_orden`, `id_cliente`, `id_equipo`, `nro_serie`, `adquirido_en`, `nro_factura`, `falla`, `reparacion`, `estado`) VALUES (:yp0, :yp1, :yp2, :yp3, :yp4, :yp5, :yp6, :yp7, :yp8)' in C:\xampp\htdocs\yii\framework\db\CDbCommand.php:357
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(1023): CDbCommand->execute()
#1 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(795): CActiveRecord->insert(NULL)
#2 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(75): CActiveRecord->save()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#10 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#11 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#13 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#14 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#15 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/create
---
2013/02/13 20:28:54 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1116): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterPostOnly(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/delete')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/sgt/index.php/ordenes/delete?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/13 20:29:32 [error] [exception.CException] exception 'CException' with message 'Propiedad "Ordenes"."id" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\db\ar\CActiveRecord.php(143): CComponent->__get('id')
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\view.php(14): CActiveRecord->__get('id')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): CController->render('view', Array)
#7 [internal function]: OrdenesController->actionView('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/index
---
2013/02/13 20:29:47 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1116): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterPostOnly(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#10 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('delete')
#12 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/delete')
#13 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#14 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/sgt/index.php/ordenes/delete?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/13 20:30:37 [error] [php] Undefined index: service_oficial (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:76)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/create
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (76)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 20:39:54 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/13 20:40:37 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/update')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenes/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/13 20:44:56 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#4 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#5 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#6 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('update')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenesso/updat...')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/ordenesso/update?nro_orden=254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/update/254
---
2013/02/13 20:47:18 [error] [php] Undefined variable: nro_orden (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:113)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/update/254
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (113)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 20:48:07 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:114)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/update/254
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (114)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 20:50:08 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:118)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/update/254
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (118)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 20:51:21 [error] [php] Trying to get property of non-object (C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php:118)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#6 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#8 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#10 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#11 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#12 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/update/254
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (118)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/13 21:08:44 [error] [php] Undefined variable: nro_orden (C:\xampp\htdocs\sgt\protected\controllers\OrdenesSoController.php:79)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesSoController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#3 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesSoController->filterAccessControl()
#4 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesSoController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesSoController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenesso/create?nro_orden=254
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesSoController.php (79)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/14 20:13:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php/site/login
---
2013/02/14 20:15:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php/site/login
---
2013/02/14 20:15:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consulta" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consulta')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consulta')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consulta
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/consulta
---
2013/02/14 20:16:19 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consultarOrden" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consultarOrden')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consultarOrden')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consultarOrden
HTTP_REFERER=http://localhost/sgt/index.php/site/index
---
2013/02/14 20:18:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'El sistema no ha podido encontrar la acción "consultarOrden" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:483
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(270): CController->missingAction('consultarOrden')
#1 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consultarOrden')
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/consult...')
#3 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#4 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/sgt/index.php/ordenes/consultarOrden
HTTP_REFERER=http://localhost/sgt/index.php/site/index
---
2013/02/14 20:18:11 [error] [exception.CHttpException.400] exception 'CHttpException' with message 'Su solicitud es inválida.' in C:\xampp\htdocs\yii\framework\web\CController.php:336
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(309): CController->invalidActionParams(Object(CInlineAction))
#1 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#2 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consultarOrden')
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/consultarO...')
#5 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#6 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#7 {main}
REQUEST_URI=/sgt/index.php/site/consultarOrden
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/consultarOrden
---
2013/02/14 20:20:04 [error] [php] Use of undefined constant nro_orden - assumed 'nro_orden' (C:\xampp\htdocs\sgt\protected\controllers\SiteController.php:115)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (115)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/18 22:55:12 [error] [php] Missing argument 2 for CActiveForm::label(), called in C:\xampp\htdocs\sgt\protected\views\site\consultaDeOrden.php on line 19 and defined (C:\xampp\htdocs\yii\framework\web\widgets\CActiveForm.php:545)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): SiteController->renderInternal()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#3 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(131): SiteController->render()
#4 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden
in C:\xampp\htdocs\sgt\protected\views\site\consultaDeOrden.php (19)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (131)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/18 22:56:43 [error] [php] Undefined variable: model_orden (C:\xampp\htdocs\sgt\protected\views\site\consultaDeOrden.php:19)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): SiteController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): SiteController->renderPartial()
#2 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(131): SiteController->render()
#3 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#4 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#9 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#10 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden
in C:\xampp\htdocs\sgt\protected\views\site\consultaDeOrden.php (19)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (131)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/18 23:08:57 [error] [php] Undefined index: password (C:\xampp\htdocs\sgt\protected\controllers\SiteController.php:131)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#2 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#3 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#4 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#5 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes%5Bnro_orden%5D=254&Clientes%5Bpassword%5D=uibiu&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (131)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/18 23:14:56 [error] [exception.CException] exception 'CException' with message 'SiteController no ha podido encontrar la vista "/site/consultarOrden" solicitada.' in C:\xampp\htdocs\yii\framework\web\CController.php:878
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('/site/consultar...', Array, true)
#1 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(133): CController->render('/site/consultar...', Array)
#2 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#5 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#6 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consultarOrden')
#7 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/consultarO...')
#8 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#9 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#10 {main}
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes%5Bnro_orden%5D=254&Clientes%5Bpassword%5D=1234&yt0=Consultar+Orden
HTTP_REFERER=http://localhost/sgt/index.php/site/consultarOrden
---
2013/02/20 01:15:01 [error] [exception.CException] exception 'CException' with message 'Propiedad "CJuiDatePicker"."0" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:173
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CWidgetFactory.php(161): CComponent->__set('0', Array)
#1 C:\xampp\htdocs\yii\framework\web\CBaseController.php(146): CWidgetFactory->createWidget(Object(CActiveForm), 'zii.widgets.jui...', Array)
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(166): CBaseController->createWidget('zii.widgets.jui...', Array)
#3 C:\xampp\htdocs\sgt\protected\views\ordenes\_form.php(190): CBaseController->widget('zii.widgets.jui...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#5 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#7 C:\xampp\htdocs\sgt\protected\views\ordenes\create.php(22): CController->renderPartial('_form', Array)
#8 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#9 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#11 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('create', Array, true)
#12 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(89): CController->render('create', Array)
#13 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): OrdenesController->actionCreate()
#14 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#16 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#18 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#19 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#20 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#21 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#22 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('create')
#23 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/create')
#24 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#25 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#26 {main}
REQUEST_URI=/sgt/index.php/ordenes/create
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/254
---
2013/02/20 21:29:10 [error] [exception.CException] exception 'CException' with message 'El alias "ext.yii-pdf.EYiiPdf" es inválido. Asegúrese de que apunta a un archivo PHP y el archivo tiene permisos de lectura.' in C:\xampp\htdocs\yii\framework\YiiBase.php:316
Stack trace:
#0 C:\xampp\htdocs\yii\framework\YiiBase.php(196): YiiBase::import('ext.yii-pdf.EYi...', true)
#1 C:\xampp\htdocs\yii\framework\base\CModule.php(386): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\yii\framework\base\CModule.php(103): CModule->getComponent('ePdf')
#3 C:\xampp\htdocs\sgt\protected\controllers\SiteController.php(119): CModule->__get('ePdf')
#4 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#5 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#6 C:\xampp\htdocs\yii\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#7 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#8 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('consultarOrden')
#9 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('site/consultarO...')
#10 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#11 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#12 {main}
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes%5Bnro_orden%5D=254&Clientes%5Bpassword%5D=1234&yt0=Consultar+Orden
HTTP_REFERER=http://localhost/sgt/index.php/site/consultarOrden
---
2013/02/20 21:42:35 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (154)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:43:56 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (154)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:44:26 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (154)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:44:27 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (154)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:44:42 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (154)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:45:27 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (154)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:48:07 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (158)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/20 21:48:16 [error] [php] Cannot modify header information - headers already sent by (output started at C:\xampp\htdocs\yii\framework\web\CController.php:793) (C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php:7066)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(49): SiteController->actionConsultarOrden()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#2 C:\xampp\htdocs\yii\framework\web\CController.php(286): SiteController->runAction()
#3 C:\xampp\htdocs\yii\framework\web\CController.php(265): SiteController->runActionWithFilters()
#4 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): SiteController->run()
#5 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#6 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#7 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/site/consultarOrden?Ordenes[nro_orden]=254&Clientes[password]=1234&yt0=Consultar+Orden
in C:\xampp\htdocs\sgt\protected\vendors\mpdf\mpdf.php (7066)
in C:\xampp\htdocs\sgt\protected\controllers\SiteController.php (158)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/21 19:53:42 [error] [php] Undefined variable: model_orden (C:\xampp\htdocs\sgt\protected\views\ordenes\view.php:59)
Stack trace:
#0 C:\xampp\htdocs\yii\framework\web\CController.php(869): OrdenesController->renderFile()
#1 C:\xampp\htdocs\yii\framework\web\CController.php(782): OrdenesController->renderPartial()
#2 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): OrdenesController->render()
#3 unknown(0): OrdenesController->actionView()
#4 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs()
#5 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CInlineAction->runWithParamsInternal()
#6 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams()
#7 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): OrdenesController->runAction()
#8 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#9 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CAccessControlFilter->filter()
#10 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): OrdenesController->filterAccessControl()
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter()
#12 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(265): OrdenesController->runActionWithFilters()
#14 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): OrdenesController->run()
#15 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController()
#16 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#17 C:\xampp\htdocs\sgt\index.php(13): CWebApplication->run()
REQUEST_URI=/sgt/index.php/ordenes/254
in C:\xampp\htdocs\sgt\protected\views\ordenes\view.php (59)
in C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php (56)
in C:\xampp\htdocs\sgt\index.php (13)
2013/02/21 20:15:48 [error] [exception.CException] exception 'CException' with message 'Propiedad "CWebApplication"."base" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('base')
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\view.php(88): CModule->__get('base')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): CController->render('view', Array)
#7 [internal function]: OrdenesController->actionView('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/update/254
---
2013/02/21 20:18:09 [error] [exception.CException] exception 'CException' with message 'Propiedad "CWebApplication"."serverName" no se encuentra definida.' in C:\xampp\htdocs\yii\framework\base\CComponent.php:130
Stack trace:
#0 C:\xampp\htdocs\yii\framework\base\CModule.php(105): CComponent->__get('serverName')
#1 C:\xampp\htdocs\sgt\protected\views\ordenes\view.php(88): CModule->__get('serverName')
#2 C:\xampp\htdocs\yii\framework\web\CBaseController.php(126): require('C:\xampp\htdocs...')
#3 C:\xampp\htdocs\yii\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\xampp\htdocs...', Array, true)
#4 C:\xampp\htdocs\yii\framework\web\CController.php(869): CBaseController->renderFile('C:\xampp\htdocs...', Array, true)
#5 C:\xampp\htdocs\yii\framework\web\CController.php(782): CController->renderPartial('view', Array, true)
#6 C:\xampp\htdocs\sgt\protected\controllers\OrdenesController.php(56): CController->render('view', Array)
#7 [internal function]: OrdenesController->actionView('254')
#8 C:\xampp\htdocs\yii\framework\web\actions\CAction.php(108): ReflectionMethod->invokeArgs(Object(OrdenesController), Array)
#9 C:\xampp\htdocs\yii\framework\web\actions\CInlineAction.php(47): CAction->runWithParamsInternal(Object(OrdenesController), Object(ReflectionMethod), Array)
#10 C:\xampp\htdocs\yii\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#11 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 C:\xampp\htdocs\yii\framework\web\filters\CFilter.php(40): CFilterChain->run()
#13 C:\xampp\htdocs\yii\framework\web\CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 C:\xampp\htdocs\yii\framework\web\filters\CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 C:\xampp\htdocs\yii\framework\web\filters\CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 C:\xampp\htdocs\yii\framework\web\CController.php(291): CFilterChain->run()
#17 C:\xampp\htdocs\yii\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(282): CController->run('view')
#19 C:\xampp\htdocs\yii\framework\web\CWebApplication.php(141): CWebApplication->runController('ordenes/view')
#20 C:\xampp\htdocs\yii\framework\base\CApplication.php(169): CWebApplication->processRequest()
#21 C:\xampp\htdocs\sgt\index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/sgt/index.php/ordenes/254
HTTP_REFERER=http://localhost/sgt/index.php/ordenes/update/254
---
